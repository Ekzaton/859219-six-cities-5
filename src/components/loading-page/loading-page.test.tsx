import React from 'react';
import {render} from '@testing-library/react';

import LoadingPage from "./loading-page";

test(`Loading page renders correctly`, () => {
  const {container} = render(
      <LoadingPage/>
  );

  expect(container).toMatchSnapshot();
});
