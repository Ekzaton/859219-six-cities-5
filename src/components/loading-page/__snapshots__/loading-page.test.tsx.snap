// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading page renders correctly 1`] = `
<div>
  <main
    class="page__main"
  >
    <h1
      class="property__name"
      style="margin-top: 45vh;"
    >
      Loading data... Please wait
    </h1>
  </main>
</div>
`;
